app/config.py

```diff
@@ -25,22 +26,24 @@ class Config():
     NOTIFY_API_CLIENT_ID = "govuk-alerts"
 
     CELERY = {
-        'broker_url': 'sqs://',
-        'broker_transport_options': {
-            'region': NOTIFY_AWS_REGION,
-            'visibility_timeout': 310,
-            'queue_name_prefix': NOTIFICATION_QUEUE_PREFIX,
-            'wait_time_seconds': 20,  # enable long polling, with a wait time of 20 seconds
+        "broker_url": "https://sqs.eu-west-2.amazonaws.com",
+        "broker_transport": "sqs",
```

Comment:
Why are we using a mixture of single and double quotes here?

Reply:
Double-quotes now used throughout